### ISSUE TITLE
[Linux] Unable to bind controller button as shortcut

### ISSUE THREAD

**Issue by gyvess** [mumble-voip/mumble#3750]
Using a MadCatz Arcade Stick, Mumble does not accept any input from this Xbox peripheral.

I'm using Mumble 1.3.0 rc2 on Manjaro. Faintly remember an issue created about this years ago but I'm unable to find it.

**Comment by keenanweaver** [comment]
Still an issue today on latest stable v1.5.634

**Comment by Krzmbrzl** [comment]
@keenanweaver are you also using Linux? Are you using X11 or Wayland?

**Comment by keenanweaver** [comment]
> @keenanweaver are you also using Linux? Are you using X11 or Wayland?

Correct, on Linux. Currently using Wayland, but the issue has been occurring for me on X11 for as long as I've used Linux (3-4 years). Not a problem on Windows.

**Comment by pbzweihander** [comment]
I need other command other than defined in RPC, so my current workaround:

1. Run mumble client in X11 with `XDG_SESSION_TYPE=x11` (because of #5257)
2. Use https://github.com/sezanzeb/input-remapper to remap controller buttons to keyboard keys
3. Bind the remapped keyboard keys as shortcuts
